src/app/pages/Settings/Settings.tsx

```diff
@@ -158,15 +159,16 @@ const Settings = memo<SettingsProps>(({ tabSlug }) => {
   return (
     <PageLayout
       pageTitle={<T id={activeTab?.titleI18nKey ?? 'settings'} />}
-      contentPadding={false}
       contentClassName="pt-4 px-4"
-      paperClassName="!bg-background"
+      dimBg
```

Comment:
`dimBg` is `true` by default, so you don't need this property specification. Just make sure that it's `false` where necessary